target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -9,7 +9,19 @@ board_config_update
 
 case "$(board_name)" in
 cisco-mx100-hw)
-	ucidef_set_interfaces_lan_wan "eth0 eth1 eth2 eth3 eth4 eth5 eth7 eth8 eth9 eth10 eth11" "eth6"
+	ucidef_set_network_device_path "mgmt" "pci0000:00/0000:00:01.2/0000:03:00.3"
+	ucidef_set_network_device_path "internet" "pci0000:00/0000:00:01.2/0000:03:00.2"
```

Comment:
Would it make sense to call this `inet`?

Reply:
`wan` is an OpenWrt bridge interface, so no. `inet` is fine. I made a justification for an 8-character limit in the commit; this does not exceed that.

Reply:
Personally I would find 'inet' confusing, related to the protocol family.

Reply:
No, sorry. What I meant was, `wan` is a default OpenWrt interface name. The difference between interfaces and netdevs in my mind is that interfaces are potentially bridges of many netdevs.

Reply:
But `wan` is also used in nearly every DSA DTS as a netdev name, so I'm not sure there's a problem using it for this purpose here too.

Reply:
It appears as `wan@eth0`, in `ip link`. Can we reproduce that here?

Reply:
Eh, OK. I've thought about it, I'm over it. Though `ip link` shows `wan@eth0` on my Linksys WRT1900AC v1, `/etc/config/network` still uses:

```txt
config interface 'wan'
	option device 'wan'
	option proto 'dhcp'
```

I think `wan` is a well-known, well-understood reference to 'ethernet port label pointing to the upstream router to the greater internet', so I fold. Fixed as of force-push 0e1ccd5a110beea61447ed294aff288a3b2b72d1.